src/main/resources/sqlscripts/setup-ecmconsumer-db.sql

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
surely this file should be '.sh' instead of '.sql'

Reply:
Yes!